lib/request-fragment.js

```diff
@@ -19,8 +19,8 @@ const requiredHeaders = {
  * @param {Object} request - HTTP request stream
  * @returns {Promise} Response from the fragment server
  */
-module.exports = function requestFragment (fragmentUrl, fragmentAttributes, request) {
-    return new Promise((resolve, reject) => {
+module.exports = (fragmentUrl, fragmentAttributes, request) =>
```

Comment:
@dazld   But wouldn't the stack trace use the name from the name of the import. E.g.:
```
const requestFragment = require('./lib/request-fragment');

```

I may be wrong but based on my understanding, with any error, the stack trace would use the name of the import i.e. `requestFragment`.